modules/ecs-service/variables.tf

```diff
@@ -101,6 +101,12 @@ variable "ecs_task_cpu" {
   default     = 256
 }
 
+variable "ecs_task_ephemeral_storage" {
+  type        = number
+  description = "The amount of ephemeral storage (in GiB) to allocate to the ECS task."
+  default     = 21
```

Comment:
Wait, how? 21 GiB? Shouldn't we give it more?

Reply:
That's the default, we'd override for clark-bundling-service